x-pack/filebeat/module/mssql/log/_meta/fields.yml

```diff
@@ -0,0 +1,10 @@
+- name: log
+  description: Common log fields
+  type: group
+  fields:
+    - name: source
+      description: Origin of the message, usually the server but it can also be a recovery process
+      type: text
+    - name: msg
```

Comment:
I don't think this one is needed as it's `message`. At least I can't see it in the ingest pipeline.

Reply:
Good catch! removed.